Source/WebCore/bindings/js/JSDOMConvertStrings.cpp

```diff
@@ -71,7 +72,23 @@ String valueToByteString(JSGlobalObject& lexicalGlobalObject, JSValue value)
     auto string = value.toWTFString(&lexicalGlobalObject);
     RETURN_IF_EXCEPTION(scope, { });
 
-    return stringToByteString(lexicalGlobalObject, scope, WTFMove(string));
+    if (UNLIKELY(throwIfInvalidByteString(lexicalGlobalObject, scope, string)))
+        return { };
+    return string;
+}
+
+AtomString valueToByteAtomString(JSC::JSGlobalObject& lexicalGlobalObject, JSC::JSValue value)
+{
+    VM& vm = lexicalGlobalObject.vm();
+    auto scope = DECLARE_THROW_SCOPE(vm);
+
+    AtomString string = value.toString(&lexicalGlobalObject)->toAtomString(&lexicalGlobalObject);
```

Comment:
Nit: could be `auto`; `toAtomString`suggests what the type will be.

Reply:
Fixed.